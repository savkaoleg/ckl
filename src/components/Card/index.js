import React from 'react'
import './index.scss'

const Card = ({ text, title }) => (
  <div className="card">
    <h2 className="title">{title}</h2>
    <p>{text}</p>
  </div>
)

export default Card
