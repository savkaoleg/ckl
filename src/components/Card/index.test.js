import React from 'react'
import expect from 'expect'
import { shallow } from 'enzyme'
import Card from './index'

describe('Card', () => {
  it('Renders correctly', () => {
    const component = shallow(<Card />)

    expect(component).toMatchSnapshot()
  })
})
