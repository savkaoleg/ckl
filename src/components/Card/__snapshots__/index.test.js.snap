// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Renders correctly 1`] = `
<div
  className="card"
>
  <h2
    className="title"
  />
  <p />
</div>
`;
